temoa/data_processing/make_graphviz.py

```
@@ -2,39 +2,39 @@
+    results_dot_fmt,
+    tech_results_dot_fmt,
 )
-from GraphVizUtil import create_text_nodes, create_text_edges, getColorConfig, processInput
+from .graphviz_util import create_text_edges, create_text_nodes, get_color_config, process_input
 
 
-class GraphvizDiagramGenerator(object):
-    def __init__(self, dbFile, scenario=None, region=None, outDir='.', verbose=1):
-        self.dbFile = dbFile
+class GraphvizDiagramGenerator:
+    def __init__(self, db_file, scenario=None, region=None, out_dir='.', verbose=1):
```

Ruff (0.14.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)